examples/nextjs-shadcn/src/app/layout.tsx

```diff
@@ -1,7 +1,33 @@
+"use client";
+
 import { ThemeProvider } from "@mod-protocol/react-ui-shadcn/dist/components/theme-provider";
+import { arbitrum, base, optimism, polygon, zora } from "viem/chains";
+import { WagmiConfig, configureChains, createConfig, mainnet } from "wagmi";
+import { publicProvider } from "wagmi/providers/public";
+
+import { RainbowKitProvider, getDefaultWallets } from "@rainbow-me/rainbowkit";
 
+import "@rainbow-me/rainbowkit/styles.css";
 import "./globals.css";
 
+const { chains, publicClient } = configureChains(
+  [mainnet, polygon, optimism, arbitrum, base, zora],
+  [publicProvider()]
+);
+
+const { connectors } = getDefaultWallets({
+  appName: "Mod Example App",
+  projectId: "YOUR_PROJECT_ID", // TODO: replace with your project id
```

Comment:
maybe as an ENV variable, and throw error if not defined. this is currently hard to spot